ts/Maps/MapPointer.ts

```diff
@@ -44,6 +44,7 @@ declare global {
             onContainerMouseWheel(e: PointerEvent): void;
         }
         interface MapPointerChart extends Chart {
+            isAnimating: boolean;
```

Comment:
Please set this flag on the MapPointer interface, so we don't pollute the omnipresent Chart object.